app/routes/content/index.js

```diff
@@ -235,6 +235,11 @@ export default Ember.Route.extend(Analytics, ResetScrollMixin, SetupSubmitContro
     },
 
     setupController(controller, model) {
+        model.get('reviewLogs').then(reviewLogs => {
+            let firstLog = reviewLogs.toArray()[0];
```

Comment:
Any reason not to use [`firstObject`](https://www.emberjs.com/api/ember/2.14/classes/Ember.Array/properties/firstObject?anchor=firstObject)?